index.html

```diff
@@ -247,18 +253,17 @@ <h1 class="about-head-section">
             </li>
           </ul>
         </div>
+        <div class="action">
+          <a class="action" href="#">
+            <button class="btn">get my resume</button>
+          </a>
+        </div>
       </div>
-
-      <div class="action">
-        <a class="action" href="#">
-          <button class="btn">get my resume</button>
-        </a>
       </div>
-    </div>
 
-    <div class="skills">
+    <div class="skills-container">
       <div class="languages">
-        <div>
+        <div class="language-title">
           languages
           <i class="fa fa-chevron-down chevron-icon fa-0.8x"></i>
         </div>
```

Comment:
- [ ] The skills container is not responsive on a screen less than 960px wider. Kindly consider fixing it so that it would be placed in the center of the screen and be responsive as opposed to being stationary on the left side without being responsive as responsive design is one of the core requirements.
your design
![zxc](https://user-images.githubusercontent.com/96683843/213251237-20761bfb-4d40-4dbc-b8ea-ad02d647a728.jpg)